addons/mail/static/src/components/emojis_list/emojis_list.js

```diff
@@ -55,24 +59,16 @@ export class EmojisPopover extends Component {
      * @param {MouseEvent} ev
      */
     _onClickEmoji(ev) {
-        this.close();
-        if (this.props.onEmojiSelection) {
-            this.props.onEmojiSelection({
-                unicode: ev.currentTarget.dataset.unicode,
-            });
-        }
+        this.emojisListView.onClickEmoji(ev);
```

Comment:
should move handler in model

Reply:
I kept it for guarding `this.emojisListView` that is needed.